evalml/tests/component_tests/test_lgbm_regressor.py

```diff
@@ -185,3 +185,18 @@ def test_regression_goss(X_y_regression):
     clf.fit(X, y)
     assert clf.parameters['bagging_freq'] == 0
     assert clf.parameters['bagging_fraction'] == 0.9
+
+
+@pytest.mark.parametrize("data_type", ['pd', 'ww'])
+def test_lightgbm_multiindex(data_type, X_y_regression, make_data_type):
```

Comment:
Ah yes! I thought that since xgboost can handle tuples that the tests wouldn't be necessary, but since it uses the _rename_column_names_to_numeric method it was impacted and I had to tweak some stuff. Thank you!!